src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -187,7 +187,7 @@ public void ezPlace() {
         computer_dinghy.end = new Coordinate(1,1);
     }
 
-    public void randFire() {
+    public void randFire() {  //should this be ranamed to hardFire to match the ezFire? -Stewart
```

Comment:
I would have done that but there is already a function called hardFire. Jiayu has been working on this part so he can change if he wants